# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#result
geometry_msgs/PoseStamped resulting_pose
